webcamd.c

```diff
@@ -741,14 +749,24 @@ a_uid(const char *s)
 int
 main(int argc, char **argv)
 {
-	const char *params = "N:Bd:f:i:M:m:S:sv:hHrU:G:D:lL:";
+	const char *params = "N:Bd:f:i:M:m:S:sv:hHrU:G:D:lL:c:";
 	char *ptr;
 	int opt;
 	int opt_valid = 0;
 
 	openlog("webcamd", LOG_NDELAY | LOG_PERROR | LOG_PID, LOG_DAEMON);
 	while ((opt = getopt(argc, argv, params)) != -1) {
 		switch (opt) {
+		case 'c':
+			ptr = optarg;
+			if (strcmp("v4l2loopback", ptr) == 0) {
```

Comment:
Probably should add an ifdef here too:
`
#ifdef CONFIG_V4L2LOOPBACK
#endif
`

Reply:
Done